stable/etcd-operator/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: CoreOS etcd-operator Helm chart for Kubernetes
 name: etcd-operator
-version: 0.10.2
+version: 0.10.3
```

Comment:
Ah looks like this is already at `0.10.3` and needs another bump https://github.com/helm/charts/blob/master/stable/etcd-operator/Chart.yaml#L4

Now that I think about it, this appears to be an additional feature, so it should probably be a MINOR bump (`0.11.0`).

Reply:
@scottrigby cool, version bumped